distribution/sign.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
```

Comment:
add some explanation about how signing tool works and/or do some of the path mangling bazel side